### DISCUSSION TITLE
Is it possible to generate schema from typescript?

### DISCUSSION THREAD

**Discussion by MinimogDev** [fastify/fastify#2472] [General]
Hey everyone 👋 

I'm currently using fastify alongside [prisma](https://github.com/prisma/prisma) and have manually created schemas. Prisma, however, exports all types associated to your database structure and most of these are same to what my fastify handlers responses are, so I was wondering if it is somehow possible to use these types to generate / instead of schema objects?

**Comment by Ethan-Arrowood** [comment]
I've done types from schema before, never schema from types. But maybe looking at projects like the `json-schema-to-typescript` and `typebox` may lead you in the right direction
